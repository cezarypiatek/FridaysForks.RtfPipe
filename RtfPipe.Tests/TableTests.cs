using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace RtfPipe.Tests;

[TestClass]
public class TableTests
{
  [TestMethod]
  public void marge_scenario_1() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_2() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_3() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_4() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_5() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_6() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_7() => this.VerifyCurrentScenario();
}
