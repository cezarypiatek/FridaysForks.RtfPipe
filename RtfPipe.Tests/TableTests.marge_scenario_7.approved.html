<div style="font-size:12pt;font-family:&quot;Times New Roman&quot;, serif;">
    <table style="font-family:Cambria, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0 0 0 0.7px;">
        <colgroup>
            <col style="width:65.4px;" />
            <col style="width:56px;" />
            <col style="width:102.8px;" />
            <col style="width:177.4px;" />
            <col style="width:65.5px;" />
            <col style="width:3.2px;" />
            <col style="width:44.7px;" />
            <col style="width:102.7px;" />
        </colgroup>
        <tr style="text-align:left;">
            <td colspan="8" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:11pt;background:#A9A9A9;border:1px solid #000000;padding:4px;">
                <strong style="font-family:&quot;Arial CE&quot;, sans-serif;">Tabela zbiorcza wyników energii pierwotnej i końcowej
                </strong>
            </td>
        </tr>
        <tr style="text-align:left;">
            <td colspan="8" style="font-family:&quot;Times New Roman&quot;, serif;text-align:center;border:1px solid #000000;padding:4px;">
                <strong>Ogrzewanie &nbsp;i wentylacja
                </strong>
            </td>
        </tr>
        <tr style="text-align:left;">
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Nr źródła
                </span>
            </td>
            <td colspan="3" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Nazwa źródła
                </span>
            </td>
            <td colspan="3" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">Q
                <sub>K,H
                </sub>kWh/rok
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">Q
                <sub>P,H
                </sub>kWh/rok
            </td>
        </tr>
        <tr style="text-align:left;">
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">1
            </td>
            <td colspan="3" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Pompa ciepła
                </span>
            </td>
            <td colspan="3" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">2884,07
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">3228,58
            </td>
        </tr>
        <tr style="text-align:left;">
            <td colspan="4" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">Suma
            </td>
            <td colspan="3" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">2884,07
            </td>
            <td style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">3228,58
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="8" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">&nbsp;
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="8" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:center;border:1px solid #000000;padding:4px;">
                <strong style="font-family:&quot;Arial CE&quot;, sans-serif;">Przygotowanie ciepłej wody
                </strong>
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Nr źródła
                </span>
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Nazwa źródła
                </span>
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">Q
                <sub>K,W
                </sub>kWh/rok
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">Q
                <sub>P,W
                </sub>kWh/rok
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">1
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Pompa ciepła
                </span>
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">657,52
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">1469,52
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="4" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">Suma
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">657,52
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">1469,52
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="8" style="font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:3pt;border:1px solid #000000;padding:4px;">&nbsp;
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="4" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">Zestawienie energii pierwotnej Q
                <sub>P
                </sub>=Q
                <sub>P,H
                </sub>+Q
                <sub>P,W
                </sub>+Q
                <sub>P,L
                </sub>
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">4698,10
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">kWh/rok
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="4" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Zestawienie energii końcowej &nbsp;EK = (Q
                </span>
                <sub>K,H
                </sub>+Q
                <sub>K,W
                </sub>) / A
                <sub>f
                </sub>
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">18,44
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">kWh/(m
                <sup>2
                </sup>•rok)
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="4" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Roczny wskaźnik obliczeniowy zapotrzebowania na nieodnawialną energię pierwotną do ogrzewania, wentylacji i przygotowania ciepłej wody oraz chłodzenia
                </span>
            </td>
            <td colspan="2" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border-top:1px solid #000000;border-right:1px solid;border-bottom:1px solid #000000;border-left:1px solid #000000;padding:4px;">
                <span style="font-family:Arial, sans-serif;">EP
                </span>
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border-top:1px solid #000000;border-right:1px solid #000000;border-bottom:1px solid #000000;border-left:1px solid;padding:4px;">
                <span style="font-family:Arial, sans-serif;">24,47
                </span>
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <span style="font-family:Arial, sans-serif;">kWh/(m
                </span>
                <sup style="font-family:Arial, sans-serif;">2
                </sup>•
                <span style="font-family:Arial, sans-serif;">rok)
                </span>
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;height:8.7px;">
            <td colspan="8" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:3pt;border:1px solid #000000;padding:4px;">&nbsp;
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="8" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border:1px solid #000000;padding:4px;">
                <strong style="font-size:11pt;">Sprawdzenie warunku na EP
                </strong>
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="2" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:11pt;border-top:1px solid #000000;border-right:1px solid;border-bottom:1px solid #000000;border-left:1px solid #000000;padding:4px;">
                <span style="font-size:10pt;">EP kWh/(m2•rok)
                </span>
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:11pt;border-top:1px solid #000000;border-right:1px solid;border-bottom:1px solid #000000;border-left:1px solid;padding:4px;">&nbsp;
            </td>
            <td colspan="2" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:11pt;border-top:1px solid #000000;border-right:1px solid;border-bottom:1px solid #000000;border-left:1px solid;padding:4px;">
                <span style="font-size:10pt;">EP
                </span>
                <sub style="font-size:10pt;">ref
                </sub>
                <span style="font-size:10pt;"> kWh/(m
                </span>
                <sup style="font-size:10pt;">2
                </sup>
                <span style="font-size:10pt;">•rok)
                </span>
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:11pt;border-top:1px solid #000000;border-right:1px solid #000000;border-bottom:1px solid #000000;border-left:1px solid;padding:4px;">
                <span style="font-size:10pt;">Uwagi
                </span>
            </td>
        </tr>
        <tr style="text-align:left;color:#FF0000;">
            <td colspan="2" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border-top:1px solid #000000;border-right:1px solid;border-bottom:1px solid #000000;border-left:1px solid #000000;padding:4px;">
                <span style="font-family:Arial, sans-serif;">24,47
                </span>
            </td>
            <td style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;font-size:11pt;border-top:1px solid #000000;border-right:1px solid;border-bottom:1px solid #000000;border-left:1px solid;padding:4px;">
                <span style="font-size:10pt;font-family:Arial, sans-serif;">&lt;=
                </span>
            </td>
            <td colspan="2" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border-top:1px solid #000000;border-right:1px solid;border-bottom:1px solid #000000;border-left:1px solid;padding:4px;">
                <span style="font-family:Arial, sans-serif;">70,00
                </span>
            </td>
            <td colspan="3" style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;text-align:justify;border-top:1px solid #000000;border-right:1px solid #000000;border-bottom:1px solid #000000;border-left:1px solid;padding:4px;">
                <span style="font-family:&quot;Arial CE&quot;, sans-serif;">Warunek spełniony
                </span>
            </td>
        </tr>
    </table>
 </div>